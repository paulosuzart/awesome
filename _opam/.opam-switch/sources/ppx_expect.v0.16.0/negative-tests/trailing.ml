open! Core

(* Example with trailing output after last [%expect] node *)

let%expect_test _ =
  print_string "hello";
  [%expect {| hello |}];
  print_string "goodbye\n"
;;

let%expect_test _ =
  print_string "foo";
  [%expect {| foo |}];
  print_string "bar"
;;

let%expect_test _ =
  print_string "hello world";
  [%expect {| hello world |}]
;;
