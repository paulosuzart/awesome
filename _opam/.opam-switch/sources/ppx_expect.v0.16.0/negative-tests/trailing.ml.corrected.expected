open! Core

(* Example with trailing output after last [%expect] node *)

let%expect_test _ =
  print_string "hello";
  [%expect {| hello |}];
  print_string "goodbye\n";
  [%expect {| goodbye |}]
;;

let%expect_test _ =
  print_string "foo";
  [%expect {| foo |}];
  print_string "bar";
  [%expect {| bar |}]
;;

let%expect_test _ =
  print_string "hello world";
  [%expect {| hello world |}]
;;
